master/master.cfg

```diff
@@ -309,7 +309,7 @@ for name, worker, buildfactory, stability in BUILDERS:
 
     pull_request_builders.append(buildername)
 
-    source = GitHub(repourl=git_url, timeout=3600, mode="full", method="clean",)
+    source = GitHub(repourl=git_url, timeout=3600, mode="full", method="fresh",)
```

Comment:
Oh. There are Git(...) and GitHub(...) calls which get also the same parameters.

Maybe we could share parameter with something like:

```
# with super cool comments here!
GIT_PARAMETER= ...

GitHub(repourl=git_url, **GIT_PARAMETERS)
```

Reply:
Yeah, SGTM

Reply:
Something like 58dc87ec723f4f6f6fb246d1189da8e29791bbd7?